serverless.yml

```diff
@@ -46,23 +50,27 @@ custom:
 provider:
   name: aws
   stage: ${opt:stage, "test"}
-  runtime: python3.7
+  runtime: python3.9
   region: us-east-1
   environment: 
     DYNAMODB_CALENDAR: ${self:custom.calendarTableName}
     AUTH0_CLIENT_ID: ${file(./secrets.json):AUTH0_CLIENT_ID}
     AUTH0_CLIENT_PUBLIC_KEY: ${file(./public_key)}
-  iamRoleStatements:
-    - Effect: Allow 
-      Action: 
-        - "dynamodb:PutItem"
-        - "dynamodb:GetItem"
-        - "dynamodb:UpdateItem"
-        - "dynamodb:DeleteItem"
-        - "dynamodb:Scan"
-        - "dynamodb:Query"
-      Resource:
-        - "arn:aws:dynamodb:${self:provider.region}:*:table/${self:custom.calendarTableName}*"
+    stage: ${self:custom.stage.${self:provider.stage}}
```

Comment:
Environment variables are usually all caps, so change stage to STAGE. In python they'll probably be loaded as a constant, which merits all caps for the name too (see [pep8](https://peps.python.org/pep-0008/#constants))